tests/SideBySide/ConnectAsync.cs

```diff
@@ -258,7 +258,7 @@ public ConnectAsync(DatabaseFixture database)
 			using (var connection = new MySqlConnection(csb.ConnectionString))
 			{
 #if BASELINE || NET45
-				await Assert.ThrowsAsync<NotImplementedException>(() => connection.OpenAsync());
+				await Assert.ThrowsAsync<MySqlException>(() => connection.OpenAsync());
```

Comment:
I would expect the "baseline" library (Oracle's Connector/NET) to continue to throw `NotImplementedException` here. 

I wonder if the `|| NET45` check is out of date, because the SideBySide project builds against `net452`, so it's probably never getting into this block.